src/content/community/releases/_index.en.md

```diff
@@ -4,6 +4,37 @@ title = "Releases"
 weight = 40
 +++
 
+## v0.9.0
+
+* The gateway Pod has been renamed from `submariner` to `submariner-gateway`.
+* The Helm charts now use Submariner's Operator to deploy and manage Submariner.
+* Broker creation is now managed by the Operator instead of `subctl`.
+* Each Submariner Pod now has its own service account with appropriate privileges.
+* For OpenShift, the Lighthouse CoreDNS server metrics are now included with the default CoreDNS metrics.
```

Comment:
Not quite, we expose the Lighthouse CoreDNS metrics in all cases but they’re not merged with any other CoreDNS metrics.

Reply:
Based on https://github.com/submariner-io/lighthouse/issues/276, I changed it to  “_The Lighthouse CoreDNS server metrics are now exposed_”, hopefully that suffices.